config/krakend/settings/frinx_frontend_settings.json

```diff
@@ -2,15 +2,11 @@
     "endpoints": [
         {
             "endpoint": "/",
-            "url_pattern": "/frinxui"
+            "url_pattern": "/"
         },
         {
             "endpoint": "/favicon.ico",
             "url_pattern": "/favicon.ico"
```

Comment:
this should not be needed

Reply:
this means that "/" will be mapped to "/" in our frinx-frontend but if someone wants different UI to be shown, he needs to put this part into different JSON (specific for that UI)

Reply:
And both "/" and "/frinxui" points to our UI in this case, but if we decide we want another UI to be at "/", only "/frinxui" will be pointing to our UI

Reply:
If its not there and we add another UI, how will it decide to which UI it should map the "/"?

Reply:
In case of multiple frontends, FRINX's favicon will be the only one used for consistency, since multiple frontends will send request to get `/favicon.ico` but all of them will be catched here, therefore always returning favicon from `frinx-frontend`